experimental/unitsconversion/src/lib.rs

```diff
@@ -22,4 +24,7 @@ pub enum ConversionError {
     /// This can happen if the units are not compatible.
     /// For example, `meter` and `foot` are compatible, but `meter` and `second` are not.
     InvalidConversion,
+
+    /// The conversion is not possible due to an internal error.
+    InternalError,
```

Comment:
issue: this should not exist, each variant should be clearly named and have an explanation how the error is triggered and how it can be avoided.

Reply:
Okay, I have an error called DataNotFound, and I do not return error. it is just for debugging.